buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -995,7 +995,8 @@ class BeamModulePlugin implements Plugin<Project> {
           'org.checkerframework.checker.nullness.NullnessChecker'
         ]
 
-        if (!parseBooleanProperty(project, 'enableCheckerFramework') && !project.jenkins.isCIBuild) {
+        // Only skip checkerframework if explicitly requested
+        if (project.hasProperty('enableCheckerFramework') && !parseBooleanProperty(project, 'enableCheckerFramework')) {
```

Comment:
nit: I don't understand the style choice here, is this a limitation of groovy? Isn't `skipCheckerFramework = project.hasProperty('enableCheckerFramework') && !parseBooleanProperty(project, 'enableCheckerFramework')` preferable?

Reply:
Yea good call of course. Classic. I don't _think_ there is any groovy weirdness. Fixed.